.cursor/rules/ci-e2e-testing.mdc

```diff
@@ -0,0 +1,381 @@
+---
+globs: .ibm/*,e2e-tests/*,docs/e2e-tests/*
+alwaysApply: false
+---
+# RHDH (Red Hat Developer Hub) Repository Context
+
+This document serves as a comprehensive starting point for LLMs working with the RHDH repository. It provides detailed information about the e2e-tests, testing infrastructure and CI/CD pipeline.
+
+## Table of Contents
+
+- [E2E Testing Framework](#e2e-testing-framework)
+- [CI/CD Infrastructure](#cicd-infrastructure)
+- [Key Dependencies and Common Issues](#key-dependencies-and-common-issues)
+- [Documentation References](#documentation-references)
+
+## E2E Testing Framework
+
+### Technology Stack
+- **Testing Framework**: Playwright with TypeScript
+- **Node.js Version**: 22
```

Comment:
Nit: Consider linking the versions mentioned here (“Node 22 / Yarn 3.8.7”) to a single source of truth (e.g., package.json)

Reply:
True, added. Thanks